### PR TITLE
🔧 Add default config for Socketio port

### PR DESCRIPTION
**Pull request by madmath03**
> Please provide enough information so that others can review your pull request:

🔧 Add default config for Socketio port

> Explain the **details** for making this change. What existing problem does the pull request solve?

This is a "_workaround_" to prevent issue frappe/frappe#10119.

> Screenshots/GIFs

### CODE REVIEW

File: images/10.x-master/alpine/Dockerfile [hunk 1]
```diff
@@ -183,6 +183,7 @@ ENV DOCKER_DB_TIMEOUT=120 \
     FRAPPE_DEFAULT_SITE= \
     FRAPPE_HTTP_PORT=80 \
     FRAPPE_WEBSERVER_PORT=80 \
+    FRAPPE_SOCKETIO_PORT=3000 \
```
Comment (PierreTecher-honhon) [inline images/10.x-master/alpine/Dockerfile:186]: Just for my understanding.
Why do we keep the default value 3000 for the dockerfile but we use the prod value for the docker-compose ?

File: README.md [hunk 2]
```diff
@@ -143,6 +143,9 @@ FRAPPE_HTTP_PORT=80
 # Default backend port. Should either be 80 (default value) or any custom port defined for the app backend.
 FRAPPE_WEBSERVER_PORT=80
 
+# Default socketion port. Should either be 3000 (default value) or 80 (usual value for prod) or any custom port defined for the socketio service.
+FRAPPE_SOCKETIO_PORT=3000
```
Comment (PierreTecher-honhon) [inline README.md:147]: Shouldn't we use the prod value for the README ?

Comment (madmath03) [reply README.md:147]: In the README, write the actual default value (defined in the Dockerfile).
The Travis-CI docker-compose tests use the standard production value.

File: images/10.x-master/alpine/Dockerfile (hunk 1 above)
Comment (madmath03) [reply images/10.x-master/alpine/Dockerfile:186]: Well, my idea was to keep the Dockerfile as close as possible to Frappe default values, but use the docker-compose as an example on how to configure a "_standard_" production environment, but now you're making me doubt...

**Pull request merged**